ESP/tools/createzip.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+Import("env")
+
+import sys
+from ntpath import basename
+from zipfile import ZipFile
+import json
+
+
+def createZip(source, target, env):
+    if (
+        sys.platform.startswith("ubuntu")
+        or sys.platform.startswith("Linux")
+        or sys.platform.startswith("linux")
+    ):
+        print("Program has been built, creating zip archive!")
+        program_path = target[0].get_abspath()
+        #print("binary path", program_path)
+        #print("FLASH_EXTRA_IMAGES: %s\n" % env["FLASH_EXTRA_IMAGES"])
+        #print("ESP32_APP_OFFSET: %s\n" % env["ESP32_APP_OFFSET"])
```

Comment:
do we need those? If not, we can probably remove them, will be a bit cleaner :D

Reply:
No, we don't. I used then for debugging - forgot to take them. Out.